forum/settings.py

```diff
@@ -65,13 +65,13 @@
 CORS_ALLOW_ALL_ORIGINS = False
 
 CORS_ORIGIN_WHITELIST = [
-    'http://localhost',
+    'http://localhost:3000',
```

Comment:
Could you whitelist `http://178.212.110.52` as well?

Reply:
Our team is currently working on this, so we`ll add it soon.